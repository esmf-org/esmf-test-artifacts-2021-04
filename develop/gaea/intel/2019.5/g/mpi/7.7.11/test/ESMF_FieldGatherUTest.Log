20210405 045816.707 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.707 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210405 045816.707 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210405 045816.707 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210405 045816.707 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210405 045816.707 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.707 INFO             PET0 Running with ESMF Version   : 8.2.0 beta snapshot
20210405 045816.707 INFO             PET0 ESMF library build date/time: "Apr  5 2021" "04:38:08"
20210405 045816.707 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g
20210405 045816.707 INFO             PET0 ESMF_COMM                   : mpi
20210405 045816.708 INFO             PET0 ESMF_MOAB                   : enabled
20210405 045816.708 INFO             PET0 ESMF_LAPACK                 : enabled
20210405 045816.708 INFO             PET0 ESMF_NETCDF                 : enabled
20210405 045816.708 INFO             PET0 ESMF_PNETCDF                : disabled
20210405 045816.708 INFO             PET0 ESMF_PIO                    : enabled
20210405 045816.708 INFO             PET0 ESMF_YAMLCPP                : enabled
20210405 045816.715 INFO             PET0  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20210405 045816.715 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210405 045816.717 INFO             PET0  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20210405 045816.718 INFO             PET0  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20210405 045816.719 INFO             PET0  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20210405 045816.720 INFO             PET0  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20210405 045816.720 INFO             PET0  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20210405 045816.720 INFO             PET0 Finalizing ESMF
20210405 045816.714 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.714 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210405 045816.714 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210405 045816.714 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210405 045816.714 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210405 045816.714 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.714 INFO             PET1 Running with ESMF Version   : 8.2.0 beta snapshot
20210405 045816.714 INFO             PET1 ESMF library build date/time: "Apr  5 2021" "04:38:08"
20210405 045816.714 INFO             PET1 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g
20210405 045816.714 INFO             PET1 ESMF_COMM                   : mpi
20210405 045816.714 INFO             PET1 ESMF_MOAB                   : enabled
20210405 045816.714 INFO             PET1 ESMF_LAPACK                 : enabled
20210405 045816.714 INFO             PET1 ESMF_NETCDF                 : enabled
20210405 045816.714 INFO             PET1 ESMF_PNETCDF                : disabled
20210405 045816.714 INFO             PET1 ESMF_PIO                    : enabled
20210405 045816.714 INFO             PET1 ESMF_YAMLCPP                : enabled
20210405 045816.715 INFO             PET1  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20210405 045816.715 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210405 045816.717 INFO             PET1  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20210405 045816.718 INFO             PET1  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20210405 045816.719 INFO             PET1  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20210405 045816.720 INFO             PET1  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20210405 045816.720 INFO             PET1  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20210405 045816.720 INFO             PET1 Finalizing ESMF
20210405 045816.707 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.707 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210405 045816.707 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210405 045816.707 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210405 045816.707 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210405 045816.707 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.707 INFO             PET2 Running with ESMF Version   : 8.2.0 beta snapshot
20210405 045816.707 INFO             PET2 ESMF library build date/time: "Apr  5 2021" "04:38:08"
20210405 045816.707 INFO             PET2 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g
20210405 045816.707 INFO             PET2 ESMF_COMM                   : mpi
20210405 045816.708 INFO             PET2 ESMF_MOAB                   : enabled
20210405 045816.708 INFO             PET2 ESMF_LAPACK                 : enabled
20210405 045816.708 INFO             PET2 ESMF_NETCDF                 : enabled
20210405 045816.708 INFO             PET2 ESMF_PNETCDF                : disabled
20210405 045816.708 INFO             PET2 ESMF_PIO                    : enabled
20210405 045816.708 INFO             PET2 ESMF_YAMLCPP                : enabled
20210405 045816.715 INFO             PET2  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20210405 045816.715 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210405 045816.717 INFO             PET2  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20210405 045816.718 INFO             PET2  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20210405 045816.719 INFO             PET2  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20210405 045816.720 INFO             PET2  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20210405 045816.720 INFO             PET2  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20210405 045816.720 INFO             PET2 Finalizing ESMF
20210405 045816.708 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.708 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210405 045816.708 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210405 045816.708 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210405 045816.708 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210405 045816.708 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210405 045816.708 INFO             PET3 Running with ESMF Version   : 8.2.0 beta snapshot
20210405 045816.708 INFO             PET3 ESMF library build date/time: "Apr  5 2021" "04:38:08"
20210405 045816.708 INFO             PET3 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g
20210405 045816.708 INFO             PET3 ESMF_COMM                   : mpi
20210405 045816.709 INFO             PET3 ESMF_MOAB                   : enabled
20210405 045816.709 INFO             PET3 ESMF_LAPACK                 : enabled
20210405 045816.709 INFO             PET3 ESMF_NETCDF                 : enabled
20210405 045816.709 INFO             PET3 ESMF_PNETCDF                : disabled
20210405 045816.709 INFO             PET3 ESMF_PIO                    : enabled
20210405 045816.709 INFO             PET3 ESMF_YAMLCPP                : enabled
20210405 045816.715 INFO             PET3  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20210405 045816.715 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210405 045816.717 INFO             PET3  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20210405 045816.718 INFO             PET3  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20210405 045816.719 INFO             PET3  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20210405 045816.720 INFO             PET3  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20210405 045816.720 INFO             PET3  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20210405 045816.720 INFO             PET3 Finalizing ESMF
