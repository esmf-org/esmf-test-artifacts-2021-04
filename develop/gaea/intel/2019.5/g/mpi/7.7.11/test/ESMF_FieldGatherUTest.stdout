build time -- 2021-04-24 00:41:30
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PET           2  Test Elapsed Time    2064.90600000000       msec.
  PET           1  Test Elapsed Time    2065.13500000000       msec.
  PET           3  Test Elapsed Time    2065.58200000000       msec.
  PET           0  Test Elapsed Time    8.29100000000001       msec.
